app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    result = []
+    for person in people:
+        person_instance = Person(person["name"], person["age"])
+        result.append(person_instance)
```

Comment:
It would look better if you could rewrite it using list comprehension.

Reply:
agree, use list comprehension